index.js

```diff
@@ -21,13 +25,22 @@ const userSchema = new mongoose.Schema({
     email: String,
     password: String,
     id: Number,
+    role:String,
 });
 
 
 const Users = new mongoose.model("User", userSchema);
 
 // Mongoose Schema - End
 
+
+// View Engine Setup
+// app.set("views",path.join(__dirname,"views"))
+
+app.set("views","views")

```

Comment:
should there be a "app.set("views",path.join(__dirname,"views"))"

Reply:
Yes, I will do it